### PR TITLE
Drop support for Python <3.8 and Django EOL versions

### PR DESCRIPTION
**Pull request by ferdinand-hoffmann**
Drop support for:
- Python 3.6 & 3.7
- Django 2.2, 3.1, & 4.0

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -28,11 +23,11 @@ classifiers =
 test_suite = runtests.run_tests
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.8
 packages = find:
 include_package_data = true
 install_requires =
-    django>=2.2
+    django>=3.2
```
Comment (RoTranDo) [inline setup.cfg:30]: When we drop django 4.0 support, wouldn't it make sense to restrict this here: `django>=3.2,!=4.0`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 A django application which can be used as a base / starting point for questionnaire functionality in your project.
-It heavily relies on django form fields & widgets and uses PostgreSQL JSON fields to store the results.
+It heavily relies on django form fields & widgets and uses JSON fields to store the results.
 
 ## Requirements
 
-- Python version 3.6+
+- Python version 3.8+
 - [Django](https://www.djangoproject.com) version 2.2+
```
Comment (RoTranDo) [inline README.md]:
```suggestion
- [Django](https://www.djangoproject.com) version 3.2+
```

**Comment by RoTranDo** [comment]
I guess, also the `travis` section in the `tox.ini` file should be updated.

File: README.md (hunk 2 above)
Comment (ferdinand-hoffmann) [reply README.md]: 534c1d1b1e976639dfde13f29a4e98ba8a7f67bd

**Comment by ferdinand-hoffmann** [comment]
> I guess, also the `travis` section in the `tox.ini` file should be updated.

i've removed it for now (it's not used) 9024fff9dc3471cfe00ddca4947a16c810a0ef55

File: setup.cfg (hunk 1 above)
Comment (ferdinand-hoffmann) [reply setup.cfg:30]: i think it's ok. we don't support 4.0 but we don't need to prevent it's usage

**Review by RoTranDo — APPROVED** [review]

**Pull request merged**